Document High Availability and Disaster Recovery in Messaging (#8212)
Particular/docs.particular.net: architecture/high-availability-and-disaster-recovery.md

#### Synchronous vs asynchronous

- **Synchronous replication** provides stronger durability and cleaner failover but increases latency and can reduce availability during partitions (the classic consistency‑versus‑availability trade‑off of CAP‑oriented designs).  
- **Asynchronous replication** improves latency and resilience to remote‑link failures but may lose recent writes or replay stale ones during failover because the standby is behind or because only part of the state is mirrored.

## Cross‑Region Latency Rules of Thumb

In multi‑region architectures, the physical distance between data centers sets the baseline network latency, which determines whether synchronous replication is viable. Typical round‑trip times (RTT) are:

| Network Path                     | Typical RTT |
|----------------------------------|------------:-
| Intra‑Region (Cross‑AZ)          | 1–5 ms      | 
| NA East - NA West                | 60–70 ms    | 
| NA East - EU West                | 80–100 ms   | 
| NA West - Asia‑Pacific           | 120–150 ms  | 
| NA East - Australia              | 200–240 ms  | 

> Synchronous replication between the US and Europe adds roughly 100 ms to each publish, which is typically unacceptable for real‑time streaming workloads.

Reviewer: Let's make the linter happy

```suggestion
Synchronous replication between North America and Europe adds roughly 100 ms to each publish, which is typically unacceptable for real‑time streaming workloads.
```
